articles/mfa/guides/mfa-api/manage.md

```diff
@@ -83,6 +83,7 @@ For the purposes of building an user interface for end users to manage their fac
 
 ::: note
 - When a user enrolls with Push, Auth0 creates an OTP enrollment. You will see both when listing enrollments.
+- If both SMS and Voice are enabled, when a user enrolls with either SMS or Voice, Auth0 will automatically create two authenticators for the phone number, one for `sms` and another for `voice`. 
```

Comment:
cc @aaguiarz is this statement still accurate? I was trying to make it more clear.

Reply:
If so we can apply to line 121 as well.

Reply:
Yes, it's accurate